### ISSUE TITLE
Column resizing in Foreign Key editor is messed up

### ISSUE THREAD

**Issue by kAlvaro** [HeidiSQL/HeidiSQL#236]
## Steps to reproduce this issue

1. Click on a table to open table editor.
2. Go to *Foreign Keys* tab and hit *Add*.
3. Drag separator between *Key name* and *Columns*.
5. Drag separator between *Columns* and *Reference table*.

## Current behavior

- When you drag separator between *Key name* and *Columns* then *Key name* and *Columns* shrink/grow alternatively, other columns remain untouched. Separator itself remains under the mouse pointer.

- When you drag separator between *Columns* and *Reference table* then *Key name* and *Columns* shrink/grow alternatively, other columns remain untouched. Separator itself moves away from the mouse pointer.

In other words, two different drag points resize the same columns. All columns to the right are affected.

## Expected behavior

Elsewhere in HeidiSQL, when you drag a separator it remains under the pointer.

## Possible solution

No idea, but it feels like a a simple typo.

## Environment

* **HeidiSQL version:** 
  

Build 5277.

* **Operating system:** 
  

Windows 10 Pro (v1709, compilation 16299.371)

**Comment by kAlvaro** [comment]
I've just realised that the *Indexes* tab behaves identically—it isn't so obvious because the're only three columns so you don't resize that often. Can it be related to the fact that column set takes 100% of with?

**Comment by ansgarbecker** [comment]
This is indeed a "feature" of VirtualTree, called auto-resize. To be exact, the header has the flag hoAutoResize, which I thought was a good idea, as the right columns are somehow resized to fit 100% if the very first column is resized. Also, the user gets all the left horizontal space for the first column. I'm not very sure if I should disable that hoAutoResize flag, as disabling that probably means we get a horizontal scrollbar suddenly.

**Comment by kAlvaro** [comment]
I see... The feature itself is not bad but the interface is horrible.

Whatever, if it's just me...

**Comment by ansgarbecker** [comment]
Was fixed some time ago. Thanks for your patience ;)

**Comment by kAlvaro** [comment]
Works fine now, thank you! (Sorry for the late report ;-)
